<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ventana_config</class>
 <widget class="QDialog" name="ventana_config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="gBox_Metodos">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Metodo de reconocimiento</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>102</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="Rdio_Eigenfaces">
       <property name="text">
        <string>Eigenfaces</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Rdio_Fisherfaces">
       <property name="text">
        <string>Fisherfaces</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Rdio_LBPH">
       <property name="text">
        <string>LBPH</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gBox_TamCaras">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>161</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Tama√±o  minimo de caras</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>108</width>
      <height>77</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="Rdio_200x200">
       <property name="text">
        <string>200 X 200 px</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Rdio_150x150">
       <property name="text">
        <string>150 X 150 px</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Rdio_100x100">
       <property name="text">
        <string>100 X 100 px</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>300</y>
     <width>174</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hLayout_Botones">
    <item>
     <widget class="QPushButton" name="Btn_Cancelar">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="Btn_Aceptar">
      <property name="text">
       <string>Guardar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
